doc/patch-author-guide.md

```diff
@@ -324,9 +305,9 @@ structure can be used by patched code to handle both new (post-patch) and
 existing (pre-patch) instances.
 
 (This example is trimmed to highlight this use-case. Boilerplate code is also
-required to allocate/free a shadow variable called "reqs_active_v2" whenever a
-new `struct kioctx` is created/released. No values are ever assigned to the
-shadow variable.)
+required to allocate/free a shadow variable with enum ID SV_REQS_ACTIVE_V2
```

Comment:
I had to do a double take at the SV_ prefix, maybe give it a KPATCH_SHADOW prefix instead?  Also it might be clearer to define it somewhere in the code.

Reply:
KPATCH_SHADOW_ prefix is kinda long.  Is something like ID_, KLP_SV, or KPS_ too cryptic?

Reply:
If it makes it more readable, why not go with the longer version?  The extra characters are free.  KLP_SV and KPS_ are cryptic, and ID_ doesn't make clear that it's a kpatch-specific id.